OSDOCS-19542 [NETOBSERV] Tags (#113088)
openshift/openshift-docs: modules/network-observability-role-based-access-control-for-loki-logs.adoc

Control who can view logs to maintain security and meet compliance requirements. By default, {logging} 5.8 and later does not grant users access to logs. Configure role-based access control to restrict access to application, infrastructure, and audit logs.

For more information on access control for Loki logs, see: "Fine grained access for Loki logs".

[id="grant-non-admin-users-cluster-wide-log-access_{context}"]
== Grant non-admin users cluster-wide log access

Add users to a custom admin group to grant cluster-wide log access without making them cluster administrators. This is useful for senior engineers who need full log visibility but should not have cluster modification privileges.

Users who are members of any group specified in the `adminGroups` field of the `LokiStack` custom resource (CR) have the same read access to logs as administrators.

.Example LokiStack CR
[source,yaml]
----
apiVersion: loki.grafana.com/v1
kind: LokiStack
metadata:
  name: loki
  namespace: netobserv-loki
spec:
  tenants:
    mode: openshift-network
    openshift:
      adminGroups:
      - cluster-admin

>       - custom-admin-group

Reviewer: @memodi @jotak was this ever been tested ? I'm discovering it 😺

Reviewer 2: yes, I remember to test this Loki feature in past

----

where:

> `spec.tenants.mode`:: Specifies the tenant mode. Set to `openshift-network` or `openshift-logging` depending on your deployment. Custom admin groups are only available in these modes.

Reviewer: ```suggestion
`spec.tenants.mode`:: Specifies the tenant mode. Must be `openshift-network` for network observability.
```
